tests/base.py

```diff
@@ -190,13 +219,16 @@ def run_and_verify_sync(self, conn_id, streams=None):
 
         return sync_record_count
 
-    def dt_to_ts(self, dtime):
-        for date_format in self.DATETIME_FMT:
-            try:
-                date_stripped = int(time.mktime(dt.strptime(dtime, date_format).timetuple()))
-                return date_stripped
-            except ValueError:
-                continue
+    def dt_to_ts(self, dtime, format):
```

Comment:
Avoid using built-in keywords as a normal argument, `format` is the built-in keyword in python.

Reply:
Done